_posts/2021-08-20-simpleblog.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "[CVE-2021-36748] Improper neutralization of SQL parameter in SimpleBlog module from Prestahome for PrestaShop"
+categories: module
+author:
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,simpleblog"
+severity: "critical (7.5)"
+---
+
+A SQL Injection issue in the list controller of the Prestahome Blog (aka ph_simpleblog) module before 1.7.8 for Prestashop allows a remote attacker to extract data from the database via the sb_category parameter.
+
+## Summary
+
+* **CVE ID**: [CVE-2021-36748](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-36748)
+* **Published at**: 2021-08-20
+* **Advisory source**: [sorcery.ie](https://blog.sorcery.ie/posts/simpleblog_sqli/)
+* **Vendor**: PrestaShop
+* **Product**: SimpleBlog
+* **Impacted release**: < 1.7.8
+* **Product author**: Prestahome
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+This blog post details an SQLi sorcery.ie found in Blog for Prestashop (ph_simpleblog) by Prestahome.
+
+## CVSS base metrics
+
+* **Attack vector**: network
+* **Attack complexity**: low
+* **Privilege required**: none
+* **User interaction**: none
+* **Scope**: unchanged
+* **Confidentiality**: high
+* **Integrity**: high
+* **Availability**: high
+
+**Vector string**: [CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H)
+
+## Possible malicious usage
+
+* Obtain admin access
+* Remove data from the associated PrestaShop
+* Copy/paste data from sensitive tables to FRONT to exposed tokens and unlock admins's ajax scripts
```

Comment:
Copy/paste data from sensitive tables to the FRONT to expose tokens and unlock admins's ajax scripts